src/subcommands/sign.rs

```diff
@@ -19,34 +21,63 @@ pub struct Sign {
 
     /// String of UTF-8 text
     input_data: String,
+
+    /// Encode the signature in ASN.1 format (for ECC signatures
+    /// only).
+    #[structopt(long)]
+    encode_asn1: bool,
```

Comment:
I'd be for removing that flag and always doing ASN1 encoding until someone needs it